src/PdfDroplet.csproj

```diff
@@ -76,20 +76,20 @@
         <PackageReference Include="GioSharp-signed" Version="3.22.24.37" />
         <PackageReference Include="GLibSharp-signed" Version="3.22.24.37" />
         <PackageReference Include="GtkSharp-signed" Version="3.22.24.37" />
-        <PackageReference Include="icu.net" Version="2.8.1" />
-        <PackageReference Include="L10NSharp" Version="7.0.0" />
-        <PackageReference Include="Markdig.Signed" Version="0.30.4" />
+        <PackageReference Include="icu.net" Version="3.0.1" />
+        <PackageReference Include="L10NSharp" Version="8.0.0" />
+        <PackageReference Include="Markdig.Signed" Version="0.37.0" />
         <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="2.0.4" />
         <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="2.0.4" />
         <PackageReference Include="Mono.Posix" Version="5.4.0.201" />
         <PackageReference Include="NDesk.DBus" Version="0.15.0" />
         <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
         <PackageReference Include="PangoSharp-signed" Version="3.22.24.37" />
-        <PackageReference Include="PdfSharp" Version="1.50.5147" />
-        <PackageReference Include="SIL.Core" Version="14.2.0-beta0019" />
-        <PackageReference Include="SIL.Core.Desktop" Version="14.2.0-beta0019" />
-        <PackageReference Include="SIL.Windows.Forms" Version="14.2.0-beta0019" />
-        <PackageReference Include="SIL.WritingSystems" Version="14.2.0-beta0019" />
+		<PackageReference Include="PdfSharp" Version="6.2.1" />
+        <PackageReference Include="SIL.Core" Version="16.1.0-*" />

```

Comment:
I will have a PR with changes to SIL.Core coming as part of this .net 8 upgrade. Though this seems to work fine without them